blockchain/blockchain-wallet-v4-frontend
# Feature/link logic

## Description
Adds ExchangeService which comes with `canTrade` function to determine which partner a user canTrade with based on their location, invite status, or account status. If function takes `(settings, options, buySell, type)` and return partner name.

Tickets: 638

## Change Type
Please enter one or more of the following: 
- Upgrades

## Testing Steps
Create an account on staging with a non @blockchain.com email, you will not be able to use the Buy/Sell feature.
Create an account on staging with a @blockchain.com email, you will be able to see the Buy/Sell feature.

## PR Creator Checklist

## PR Reviewer Checklist
